concourse/model/job.py

```diff
@@ -23,6 +24,11 @@
 from concourse.model.resources import RepositoryConfig, ResourceIdentifier
 
 
+class AbortObsoleteJobs(enum.Enum):
```

Comment:
in fact, I also would prefer to use `obsolete` (in the PR-description's example, you named the attr `outdated` - let us please make this consistent)